keyboards/planck/keymaps/vifon/keymap.c

```diff
@@ -89,28 +91,46 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     {_______, _______, KC_LGUI, KC_LALT, _______, _______,     _______,        _______, _______, _______, _______, _______}
 },
 [_DYN]= { /* special */
-    {_______,  DYN_REC_START1, DYN_MACRO_PLAY1, _______, _______, _______, _______, KC_APP,  KC_INS,  _______, KC_PSCR, KC_PAUS},
-    {_______,  DYN_REC_START2, DYN_MACRO_PLAY2, _______, _______, _______, _______, _______, _______, KC_CAPS, KC_SLCK, KC_NLCK},
-    {KM_SHLK,  _______,        _______,         _______, _______, _______, _______, _______, _______, _______, _______, _______},
-    {_______,  _______,        _______,         _______, _______, _______, _______, _______, _______, _______, _______, _______}
+    {KM_HOLD,  DYN_REC_START1, DYN_MACRO_PLAY1, _______, _______,    _______, _______, KC_APP,  KC_INS,  _______, KC_PSCR, KC_PAUS},
+    {KC_LOCK,  DYN_REC_START2, DYN_MACRO_PLAY2, _______, _______,    _______, _______, _______, _______, KC_CAPS, KC_SLCK, KC_NLCK},
+    {KM_SHLK,  _______,        _______,         _______, _______,    _______, _______, _______, _______, _______, _______, _______},
+    {_______,  _______,        _______,         _______, KM_PP_GAME, DF(_QW), DF(_QW), DF(_PP), _______, _______, _______, _______}
 },
 };
 
 #undef _______
+#undef XXXXXXX
 
 const uint16_t PROGMEM fn_actions[] = {
     ACTION_MODS_TAP_KEY(MOD_RSFT, KC_ENT),
+    ACTION_MODS_TAP_KEY(MOD_RSFT, KC_DOT),
+    ACTION_MODS_TAP_KEY(MOD_RCTL, KC_SLSH),
 };
 
+/* It's a pseudo-layer composed of two real layers, we need a function for this. */
+void enable_gaming_layer(void) {
```

Comment:
you could actually use `default_layer_state_set_user` here, the same way that `layer_state_set_user` works, and just use `DF(x)` in your keymap. Or use the `default_layer_set` command. 

Optional, but ..

Reply:
It's basically a wrapped `default_layer_set` call already. Since I also add blinking with LEDs I'd rather keep it this way. Thanks for the suggestion though!